connectors/src/api/createConnector.ts

```diff
@@ -36,7 +36,8 @@ const _createConnectorAPIHandler = async (
       return apiError(req, res, {
         api_error: {
           type: "invalid_request_error",
-          message: `Missing required parameters. Required : workspaceAPIKey, dataSourceName, workspaceId, nangoConnectionId`,
+          message: `Missing required parameters. Required : workspaceAPIKey,
```

Comment:
nit: the newline will get included in the string

Reply:
Yes we need to find a standardized way to handle these that we feel great about it. For now long lines it is.

Reply:
Alternatively we can use `+` to concatenate lines

Reply:
Yes I think that's the only option... but I wanted to triple check before setting up a policy (rule?) for that.